packages/shared/src/components/modals/squads/SquadNotificationsModal.tsx

```diff
@@ -143,12 +144,9 @@ export function SquadNotificationsModal({
         )}
         <p className="text-text-tertiary typo-callout">
           For more options, go to your
-          <a
-            className="ml-1 text-text-link underline hover:no-underline"
-            href="/account/notifications"
-          >
+          <Anchor className="ml-1" href="/account/notifications">
```

Comment:
Probably also safer to use full URL here?

Reply:
Yeah, let's update this one too.